frontend/src/components/GifWidget/GifWidget.tsx

```diff
@@ -0,0 +1,38 @@
+import useGifWidget from "components/GifWidget/useGifWidget";
+import {
+    GifWidgetWrapper,
+    GifWidgetSearch,
+    GifList,
+    GifWidgetFooter
+} from "./GifWidget.styled";
+import GiphyAttributionLogoImage from "assets/GiphyAttributionLogo.png";
+
+type TGifWidget = { toggleGifWidgetVisibility: () => void };
+
+const GifWidget = ({ toggleGifWidgetVisibility }: TGifWidget) => {
+    const { gifsIsLoading, fetchGifs, findGif, gifSearchInput, ref } =
+        useGifWidget(toggleGifWidgetVisibility);
+
+    return (
+        <GifWidgetWrapper ref={ref}>
+            <GifWidgetSearch
+                type="search"
+                value={gifSearchInput}
+                onChange={findGif}
+                autoFocus={true}
+                placeholder="Znajdź gifa"
+            />
+            <GifList>{!gifsIsLoading && fetchGifs()}</GifList>
+            <GifWidgetFooter>
+                <span>Powered by</span>
+                <img
+                    style={{ width: "5rem" }}
```

Comment:
Get rid of inline style

Reply:
This is only style which this img contain, i think this is good idea, i know to hold styles into css, but for that i think it's good, we have on users list loading also styled like this, if i must refactor that, then u also must refactor on users list, for hold every style into css files, in our case in styled component

Reply:
I will do it.